package.json

```diff
@@ -19,6 +19,7 @@
     "@sendgrid/mail": "^7.3.0",
     "bcrypt": "^5.0.0",
```

Comment:
I think this dependency should be `bcryptjs` from @jcypret's commit [here](https://github.com/ualr-htcpcp/dcia/commit/ec9165f06d205c94dba0e89d64c39dbb894eb815#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519). 

Was this an issue with faker?

Also it looks like we lost the correct `"start"` command at line 11. Should be: `"start": "next start -p ${PORT:-3000}"` to work with Heroku and Cypress. Re: [this commit](https://github.com/ualr-htcpcp/dcia/commit/67ad7eae3c8e1f613073fc9b777134bead60ffc7#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)

Reply:
I am not sure what caused the alterations here. After installing faker I had to reinstall cypress and node for some reason. I belive that caused some unwanted changes.